### PR TITLE
LPS-126047 Support Content Performance Panel in Widget Pages

### PR DESCRIPTION
**Pull request by cgoncas**
**Motivation**

As for 7.3 we've developed what can be considered an MVP for content performance (metrics for display page templates of web content), now we are aiming to retrieve and display these metrics for all kinds of pages: DPT for all assets but also content and widget pages.

**Proposed Solution**

1. The Content Performance icon will be visible for Widget Pages.
2. The languages listed in the flag selector will be order alphabetically (with the default language on top) and no translated label will be seen.

**How to test it**

1. Connect to AC 
2. Create a Widget Page
3. The Content Performance icon should be visible
4. The languages in the flag selector should be alphabetically sorted (with the default language on top) and no translated label will be seen.

Screenshot 2021-03-02 at 16 31 48

### CODE REVIEW

**Comment by cgoncas** [comment]
ci:test:sf

**Comment by cgoncas** [comment]
ci:test:relevant

**Comment by cgoncas** [comment]
ci:test:stop

**Comment by cgoncas** [comment]
ci:stop

**Comment by cgoncas** [comment]
ci:test:sf

**Comment by cgoncas** [comment]
ci:test:relevant

File: modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/META-INF/resources/js/components/Translation.js [hunk 1]
```diff
@@ -86,23 +86,13 @@ export default function Translation({
...
-											}
-										>
-											{language.default
-												? Liferay.Language.get(
-														'default'
-												  )
-												: Liferay.Language.get(
-														'translated'
-												  )}
+									{language.default ? (
```
Comment (dgarciasarai) [inline modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/META-INF/resources/js/components/Translation.js:89]: You don't need a ternary here 😄   You can just do:

```js
{language.default && (
	<ClayLabel displayType="primary">
		{Liferay.Language.get('default')}
	</ClayLabel>
)}
```

**Review by dgarciasarai — CHANGES_REQUESTED** [review]
Hey @cgoncas, don't use a ternary, just a simple block like it was an `if`. See my comment with the proper way to do it. Thanks! 👍

File: modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/META-INF/resources/js/components/Translation.js (hunk 1 above)
Comment (cgoncas) [reply modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/META-INF/resources/js/components/Translation.js:89]: Ohhh!!!, i like this approach :). Awesome!!

**Comment by cgoncas** [comment]
ci:stop

File: modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/META-INF/resources/js/components/Translation.js (hunk 1 above)
Comment (cgoncas) [reply modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/META-INF/resources/js/components/Translation.js]: I have applied your feedback in https://github.com/liferay-tango/liferay-portal/pull/959/commits/f17cc40c243cc69cf7264be1770facf82b371b83

**Comment by cgoncas** [comment]
ci:test:sf

**Comment by cgoncas** [comment]
ci:test:relevant

**Review by dgarciasarai — APPROVED** [review]
Thanks @cgoncas, now the frontend part is correct! ✅

**Comment by cgoncas** [comment]
> Thanks @cgoncas, now the frontend part is correct! ✅

Thank you very much @dgarciasarai

**Comment by cgoncas** [comment]
ci:forward

**Comment by cgoncas** [comment]
ci:test:relevant

**Comment by cgoncas** [comment]
ci:test:sf

**Comment by cgoncas** [comment]
ci:test:relevant

**Comment by cgoncas** [comment]
ci:reevaluate:1077401_6809

**Comment by cgoncas** [comment]
ci:test:relevant

**Comment by cgoncas** [comment]
ci:forward

**Pull request closed without merging**